docs/UserGuide.md

```diff
@@ -51,7 +52,9 @@ Here are some example commands you can try:
 **:information_source: Notes about the command format:**<br>
 
 * Words in upper case are the parameters to be supplied by you for their respective fields.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter for the name field `n`, which can be used as `add n/Jason`.
+  e.g. in `add n/NAME`, `NAME` is a parameter for the name field `n`, which can be used as `add n/Jason`.<br>
+:bulb: **Tip:** If you are not sure what specific parameter to supply for any of the required fields, supply an estimate or random value as place holder.<br>
+:bulb: **Tip:** For any fields with an unspecified maximum length, the display may look untidy if you supply a long parameter.
```

Comment:
more like a note.
maybe can say that the list view will be truncated for fields with long parameters

Reply:
will ammend